CI build fails with GLTFAST_KEEP_MESH_DATA

**Describe the bug**
Recently our demands changed and we needed to enable `GLTFAST_KEEP_MESH_DATA`. This seems to work as expected when building locally, but in our CI (github actions) we receive an error:

```
VERBOSE: Performing the operation "System.Diagnostics.Process.Start()" on target "C:\Program Files\Unity\Hub\Editor\2020.3.15f2\Editor\Unity.exe -executeMethod NugetForUnity.NugetHelper.Restore -logFile ".\Logs\LogNuGet.log" -buildTarget WSAPlayer -batchmode -quit -verbose -nographics -logDirectory Logs -projectPath "D:\a\twinbuild.unity\twinbuild.unity"".
VERBOSE: Checking .\Logs\LogNuGet.log for errors
Write-Error: Errors were found in .\Logs\LogNuGet.log:
```

The errors it produces in the logs are:
```
Library\PackageCache\com.atteneder.gltfast@66912b2257\Runtime\Scripts\PrimitiveDracoCreateContext.cs(70,13): error
CS0103: The name 'Profiler' does not exist in the current context
Library\PackageCache\com.atteneder.gltfast@66912b2257\Runtime\Scripts\PrimitiveDracoCreateContext.cs(72,13): error
CS0103: The name 'Profiler' does not exist in the current context [Script Updater]
D:\a\twinbuild.unity\twinbuild.unity\Library\PackageCache\com.atteneder.gltfast@66912b2257\Runtime\Scripts\PrimitiveDrac
oCreateContext.cs(70,13): error CS0103: The name 'Profiler' does not exist in the current context [Script Updater]
D:\a\twinbuild.unity\twinbuild.unity\Library\PackageCache\com.atteneder.gltfast@66912b2257\Runtime\Scripts\PrimitiveDrac
oCreateContext.cs(72,13): error CS0103: The name 'Profiler' does not exist in the current context
```

For some reason Unity also seems to expand this section to it's full namespace declaration (this occurs locally too). In the source code it is:

```
#if GLTFAST_KEEP_MESH_DATA
            Profiler.BeginSample("UploadMeshData");
            mesh.UploadMeshData(false);
            Profiler.EndSample();
#else
            /// Don't upload explicitely. Unity takes care of upload on demand/deferred

            // Profiler.BeginSample("UploadMeshData");
            // mesh.UploadMeshData(true);
            // Profiler.EndSample();
#endif
```

But when opened with Unity it is:
```
#if GLTFAST_KEEP_MESH_DATA
            UnityEngine.Profiling.Profiler.BeginSample("UploadMeshData");
            mesh.UploadMeshData(false);
            UnityEngine.Profiling.Profiler.EndSample();
#else
            /// Don't upload explicitely. Unity takes care of upload on demand/deferred

            // Profiler.BeginSample("UploadMeshData");
            // mesh.UploadMeshData(true);
            // Profiler.EndSample();
#endif

```

**Desktop (please complete the following information):**
 - glTFast version `3.3.0`
 - Unity Editor version `2020.3.15f2`
 - Render Pipeline and version [e.g. Universal Render Pipeline 12.0]
 - Platform: Windows Universal ARM64

**Ideas**

It looks like commenting out the code may have resulted in the `UnityEngine.Profiling` namespace being ommitted from https://github.com/atteneder/glTFast/blob/main/Runtime/Scripts/PrimitiveDracoCreateContext.cs

--- a/Runtime/Scripts/PrimitiveDracoCreateContext.cs
+++ b/Runtime/Scripts/PrimitiveDracoCreateContext.cs
@@ -67,9 +67,9 @@ namespace GLTFast {
             }
 
 #if GLTFAST_KEEP_MESH_DATA
-            Profiler.BeginSample("UploadMeshData");
+            UnityEngine.Profiling.Profiler.BeginSample("UploadMeshData");
             mesh.UploadMeshData(false);
-            Profiler.EndSample();
+            UnityEngine.Profiling.Profiler.EndSample();
 #else
             /// Don't upload explicitely. Unity takes care of upload on demand/deferred
 

